weldx/schemas/weldx.bam.de/weldx/core/geometry/spatial_data-0.1.1.yaml

```diff
@@ -70,6 +73,8 @@ properties:
         wx_shape: [(~), 3]
       - tag: "asdf://weldx.bam.de/weldx/tags/core/data_array-0.1.*"
         wx_shape: [..., 3]
+      - tag: "asdf://weldx.bam.de/weldx/tags/units/quantity-0.1.*"
+        wx_shape: [..., 3]
```

Comment:
we probably need the `wx_units` validator here as well? (and for the `data_array`)

Reply:
If we tell the `wx_unit` validator where to look for the `units` property in the tree (it should be `node["coordinates"]["data"]` here) it should work:
```yaml
      - tag: "asdf://weldx.bam.de/weldx/tags/core/data_array-0.1.*"
        wx_shape: [..., 3]
        wx_unit:
          coordinates:
            data: "m"
```